docs/configuration/http_filters/router_filter.rst

```diff
@@ -50,7 +50,7 @@ x-envoy-max-retries
 
 If a retry policy is in place, setting this header controls the number of retries that Envoy will
 perform on the request's behalf. If the header is not specified and there is no :ref:`route
-configuration <config_http_conn_man_route_table_route_retry>` Envoy will perform a single retry. A
+configuration <config_http_conn_man_route_table_route_retry>` Envoy will NOT attempt any retries. A
```

Comment:
Not quite, basically if x-envoy-retry-on semantics are specified either via x-envoy-retry-on, or a route retry policy, the default is 1 retry. x-envoy-max-retries can modify that. If there is no retry policy specified, there are no retries no matter what x-envoy-max-retries is set to.